inscriptions/ISic000560.xml

```diff
@@ -191,56 +196,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR138337" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la">
                 <ab>
-                    <lb n="1"/>
-                    <expan>
-                        <abbr>D</abbr>
-                        <ex>is</ex>
-                    </expan>
-                    <expan>
-                        <abbr>M</abbr>
-                        <ex>anibus</ex>
-                    </expan>
-                    .<lb n="2"/>Fortis
-                    <expan>
-                        <abbr>Caesar</abbr>
-                        <ex>is</ex>
-                    </expan>
-                    <expan>
-                        <abbr>ser</abbr>
-                        <ex>vus</ex>
-                    </expan>
-                    <lb n="3"/>sibi et Charidi<lb n="4"/>coniugi bene<lb n="5"/>merenti et
-                    <expan>
-                        <abbr>P</abbr>
-                        <ex>ublio</ex>
-                    </expan>
-                    Aelio
-                    <lb n="6"/>
-                    <expan>
-                        <abbr>Aug</abbr>
-                        <ex>usti</ex>
-                    </expan>
-                    <expan>
-                        <abbr>lib</abbr>
-                        <ex>erto</ex>
-                    </expan>
-                    Felici poste
-                    <lb n="7" break="no"/>
-                    <expan>
-                        <abbr>risq</abbr>
-                        <ex>ue</ex>
-                    </expan>
-                    suorum fecit.
+                    <lb n="1"/><expan><abbr>d</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>anibus</ex></expan>
+                    <lb n="2"/>fortis <g ref="#interpunct">·</g> <expan><abbr>Caesar</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>ser</abbr><ex>vus</ex></expan>
```

Comment:
capitalise Fortis, which is attested as a slave name., and should be understood thus in the nominative as 'Fortis'